modules/ROOT/pages/kernel-args.adoc

```diff
@@ -17,26 +56,61 @@ passwd:
         - $pubkey
 systemd:
   units:
-    - name: cgroups-v2-karg.service
+    - name: cgroups-v1-karg.service
       enabled: true
       contents: |
         [Unit]
-        Description=Switch To cgroups v2
+        Description=Switch back to cgroups v1
         # We run after `systemd-machine-id-commit.service` to ensure that
         # `ConditionFirstBoot=true` services won't rerun on the next boot.
         After=systemd-machine-id-commit.service
-        ConditionKernelCommandLine=systemd.unified_cgroup_hierarchy
-        ConditionPathExists=!/var/lib/cgroups-v2-karg.stamp
+        ConditionKernelCommandLine=!systemd.unified_cgroup_hierarchy
+        ConditionPathExists=!/var/lib/cgroups-v1-karg.stamp
 
         [Service]
         Type=oneshot
         RemainAfterExit=yes
-        ExecStart=/bin/rpm-ostree kargs --delete=systemd.unified_cgroup_hierarchy
-        ExecStart=/bin/touch /var/lib/cgroups-v2-karg.stamp
+        ExecStart=/bin/rpm-ostree kargs --append=systemd.unified_cgroup_hierarchy=0
+        ExecStart=/bin/touch /var/lib/cgroups-v1-karg.stamp
         ExecStart=/bin/systemctl --no-block reboot
 
         [Install]
         WantedBy=multi-user.target
 ----
 
-NOTE:  The `After=systemd-machine-id-commit.service` directive is important to avoid some subtle issues. Similarly, any `ConditionFirstBoot=true` services should use `Before=first-boot-complete.target systemd-machine-id-commit.service`. See https://github.com/systemd/systemd/blob/3045c416e1cbbd8ab40577790522217fd1b9cb3b/man/systemd.unit.xml#L1315[the systemd documentation] for more details.
+=== Example: Disabling all CPU vulnerability mitigations
+
+Here's an example which disables all CPU vulnerability mitigations:
```

Comment:
```suggestion
Here's an example which switches `mitigations=auto,nosmt` to `mitigations=off` to disable all CPU vulnerability mitigations:
```

The reason for spelling it out more is because the formatting for `rpm-ostree kargs replace` is a bit awkward and not 100% intuitive what is getting replaced with what.